rxbinding-support-v4/src/main/java/com/jakewharton/rxbinding/support/v4/widget/NestedScrollViewScrollChangeEventOnSubscribe.java

```diff
@@ -0,0 +1,45 @@
+package com.jakewharton.rxbinding.support.v4.widget;
```

Comment:
Oh, nevermind. I haven't added them here yet. Can't keep track of all these
project.

On Mon, May 16, 2016 at 12:29 PM Christopher Kocel <notifications@github.com>
wrote:

> In
> rxbinding-support-v4/src/main/java/com/jakewharton/rxbinding/support/v4/widget/NestedScrollViewScrollChangeEventOnSubscribe.java
> <https://github.com/JakeWharton/RxBinding/pull/245#discussion_r63410356>:
>
> > @@ -0,0 +1,45 @@
> > +package com.jakewharton.rxbinding.support.v4.widget;
>
> what kind of licence? I don't see any licences in other files...
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/JakeWharton/RxBinding/pull/245/files/044fecd4b12d1017c0828387fda6ae742e5e20e1#r63410356>
>